src/app/api/pdfMonkey/pdfMonkey.api.ts

```diff
@@ -0,0 +1,70 @@
+import { PDFMONKEY_DOCUMENT_ID } from "@/models/constants";
+import { Booking, Boat } from "@/models/models";
+import moment from "moment";
+
+const pdfMonkey_api_key = process.env.PDFMONKEY_API_KEY
+
+export async function createDocument(bookingInfo: Booking, boatDetails: Boat) {
+  try {
+    console.log(">>>>>>>bookingInfo", bookingInfo)
+    console.log(">>>>>>>boatDetails", boatDetails)
+    const date = bookingInfo["Date"];
+    const bookingDateYear = moment(date).format("YYYY");
+    const bookingDateMonth = moment(date).format("MM");
+    const bookingDateDay = moment(date).format("HH");
+    const {Nombre, Code, RegistrationPlate} = boatDetails
+    const apiUrl = "https://api.pdfmonkey.io/api/v1/documents"
+
+    const response = await fetch(apiUrl, {
+      method: "POST",
+      headers: {
+        "Accept": "application/json",
+        "Content-Type": "application/json",
+        "Authorization": `Bearer ${pdfMonkey_api_key}`
+      },
+      body: JSON.stringify({
+        document: {
+          document_template_id: PDFMONKEY_DOCUMENT_ID,
+          payload: {
+            boatName: Nombre,
+            boatCode: Code,
```

Comment:
The problem here is that you are finding the PDF monkey API only for this type of booking.

The right would do this document object in the client, and send directly to the API the document object built